proprietary/tokens/src/components/amsterdam/grid.tokens.json

```diff
@@ -3,13 +3,20 @@
     "grid": {
       "column-count": { "value": "4" },
       "gap": {
-        "value": "clamp(1rem, calc(3.125vw + 0.375rem), 3.5rem)",
-        "comment": "Grows from 16px at 320px wide to 56px at 1600px wide."
+        "xs": { "value": "amsterdam.space.xs" },
+        "sm": { "value": "amsterdam.space.sm" },
+        "md": { "value": "amsterdam.space.md" },
+        "lg": { "value": "amsterdam.space.lg" },
+        "xl": { "value": "amsterdam.space.xl" }
       },
-      "padding-inline": {
-        "value": "clamp(1.5rem, calc(4.6875vw + 0.5625rem), 5.25rem)",
-        "comment": "Equals 1.5 times the gap."
+      "padding-block": {
+        "xs": { "value": "amsterdam.space.xs" },
+        "sm": { "value": "amsterdam.space.sm" },
+        "md": { "value": "amsterdam.space.md" },
+        "lg": { "value": "amsterdam.space.lg" },
+        "xl": { "value": "amsterdam.space.xl" }
       },
```

Comment:
I'm not getting this, all the grid.gap xs - xl values are stored in the mobile (4 cols) section of the grid and medium/wide do not hold any of those values. I think the hierarchy does not make sense. I would expect grid>gap to hold 1 value, and the grid>medium>gap etc.

Reply:
I understand how it looks like that, but it’s really not different from how we define variations for other components.

In this context of a grid, it reflects our thinking ‘mobile first’. All default values are in the `amsterdam.grid` namespace, including the 4 columns it initially has. In the `medium` and `wide` variations, only the value for `column-count` needs to change; the options for `gap` and `padding-*` remain the same.

We could move `"column-count": { "value": "4" },` into a `"narrow"` object to clarify this, but I think it would deviate from our regular approach.